src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -317,6 +317,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.IdentityModel.Abstractions" Version="6.22.0" />
+    <!-- The floating version exlcudes previews, see https://learn.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges -->
+    <PackageReference Include="Microsoft.IdentityModel.Abstractions" Version="6.*" />
```

Comment:
In the context of @pmaytak comment, minimum version can be specified as per this specification. https://learn.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges